src/interfaces/HposInterface.js

```diff
@@ -78,7 +91,37 @@ const HposInterface = {
     }
 
     return true
-  }
+  },
+
+  updateSettings: async ({ hostPubKey, deviceName, sshAccess }) => {
+    const settingsResponse = await hposCall({ method: 'get', path: 'config' })()
+
+    // Updating the config endpoint requires the hash of the current config to make sure nothing has changed.
+    const headers = {
+      'X-Hpos-Admin-CAS': await hashString(stringify(settingsResponse))
+    }
+
+    // settingsConfig must contain .admin.{email,public_key}, but may contain other arbitrary
+    // data.
+    const settingsConfig = {
+      ...settingsResponse
+    }
```

Comment:
What is the point of this variable? Why not mutate `settingsResponse` directly?

Reply:
The reason is to make it explicit that we are re-purposing the response and passing it as a param to the update call. Yes it's not maximally terse, but it arguably communicates what's happening a little more clearly. I don't feel strongly about this, and could go either way.

Reply:
K!